token-lending/program/tests/borrow_obligation_liquidity.rs

```diff
@@ -20,7 +20,7 @@ use spl_token_lending::{
 use std::u64;
 
 #[tokio::test]
-async fn test_borrow_quote_currency() {
+async fn test_borrow_usdc_fixed_amount() {
```

Comment:
There is no SPL Lending Market quote currency, so all tests that use USDC have been changed accordingly.